File: Herencia/ejercicio 5/Java/Empleado.java
class Empleado {
    public String nombre, apellido;
    public double salario;
    public int añosAnti;
    public Empleado(String nombre, String apellido, double salario, int añosAnti) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.salario = salario;
        this.añosAnti = añosAnti;
    }
    public double calcularSalario() {
        return salario + (salario * 0.05 * añosAnti);
    }
    public String getNombre() {
        return nombre;
    }
}

File: Herencia/ejercicio 5/Python/Empleado.py
class Empleado:
    def __init__(self, nombre, apellido, salarioBase, añosDeAntiguedad):
        self.nombre = nombre
        self.apellido = apellido
        self.salarioBase = salarioBase
        self.añosDeAntiguedad = añosDeAntiguedad
    def calcular(self):
        b = self.salarioBase * 0.05 * self.añosDeAntiguedad
        return self.salarioBase + b